src/components/pages/AdminDashboard/DevTools/DayComponent.js

```diff
@@ -17,6 +17,19 @@ const DayComponent = ({ day, devMode, setDevMode }) => {
     setDevMode(!devMode.isDevModeActive);
   };
 
+  const findNextDayOfWeek = selectedDay => {
+    let dayOfWeek = selectedDay;
+    let date = new Date();
+    let resultDate = new Date(date.getTime());
+    resultDate.setDate(date.getDate() + ((7 + dayOfWeek - date.getDay()) % 7));
```

Comment:
I think we should put a comment that this `setDate` is not the same as our `setDate` action, but is actually a JavaScript `Date()` method. Pretty unfortunate they're named the same.  Maybe we could refactor the action to have a more accurate name in the future, i.e. setCustomDate

Reply:
Yeah, I'll put a comment. The only other option would be to change the setDate function name. It shouldn't lead to any bugs cause .setDate will always be preceded by a date object, but I can see why its confusing.